lib/rbi/loc.rb

```diff
@@ -5,6 +5,21 @@ module RBI
   class Loc
     extend T::Sig
 
+    class << self
+      extend T::Sig
+
+      sig { params(file: String, yarp_location: YARP::Location).returns(Loc) }
+      def from_yarp(file, yarp_location)
+        new(
+          file: file,
+          begin_line: yarp_location.start_line,
+          end_line: yarp_location.end_line,
+          begin_column: yarp_location.start_column,
+          end_column: yarp_location.end_column + 1, # TODO: Why is this off by one?
```

Comment:
Known issue, I'm verifying Kevin's suggestion for a fix [here](https://github.com/ruby/yarp/compare/main...andyw8:yarp:andyw8/adjust-column-behaviour).

Reply:
Great! I'm going to merge and update this once fixed 👍